src/replay.ts

```diff
@@ -92,7 +95,7 @@ async function run() {
 
   // Run again after 10sec
   await snapshot.utils.sleep(10e3);
-  return run();
+  await run();
```

Comment:
Do we need `await`? 🤔

Reply:
I don't know, I trying to keep the same signature as https://github.com/snapshot-labs/snapshot-webhook/pull/51/files#diff-7dd2fefe172bde28c172c523c27b1520acf1409f9160cceb6d736056fc402626R143

Reply:
Yea I think we can remove these `await`